vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -1969,6 +2007,27 @@ func TestStandardExecutorRemoveSubnetValidatorTx(t *testing.T) {
 			},
 			expectedErr: ErrFlowCheckFailed,
 		},
+		{
+			name: "attempted to remove subnet validator after subnet manager is set",
```

Comment:
I'd break out this test and do the series of events that can lead to the `expectedErr`. The main reason (afaict) it's pretty annoying to avoid mocks is because the `state` pkg is so unwieldy. I imagine a lot of this will be streamlined once we remove historical execution.